modular_skyrat/modules/customization/modules/mob/living/carbon/human/species.dm

```diff
@@ -276,12 +276,12 @@ GLOBAL_LIST_EMPTY(customizable_races)
 /datum/species/human/felinid
 	mutant_bodyparts = list()
 	default_mutant_bodyparts = list("tail" = "Cat", "ears" = "Cat")
-	learnable_languages = list(/datum/language/common, /datum/language/nekomimetic)
+	learnable_languages = list(/datum/language/yangyu)
```

Comment:
not really a fan of this, there should be a proper replacement if you want to remove a species language, especially for a species this popular. If yangyu wasn't also a core language, I'd be more fine with it.
Species languages are something I think are quite important.

Reply:
To be undone next commit!

Reply:
Turns out this one's still up in the air, cause Xyel isn't a huge fan of neko being a thing, and there'll be Tajaran being a core language-